sandwich/src/index.ts

```diff
@@ -53,14 +54,13 @@ function provideHandleTransaction(
     // extract all swap events
     for (let i in txs) {
       const data = await web3.eth.getTransaction(txs[i]);
-
-      const decodedData = abiDecoder.decodeMethod(data.input);
+      let ID = (data.input).slice(0,10);
 
       if (
-        data.to === contractAddress &&
-        decodedData != undefined &&
-        decodedData.name === "swapTokensForExactTokens"
+        Method_ID.includes(ID) &&
```

Comment:
I would prefer to here an equality check and get rid of the array because there is no need to use `array.includes` when the array contains only one element.

Reply:
I used the array.includes because I was looking for two method Ids (swapTokensForExactTokens 0x8803dbee and swapExactTokensForTokens 0x38ed1739). But then I found out when back running the victim transaction, the bots prefered the first one and it was also implemented in the logic. The includes or regular expression method can be used when multiple method Ids are to be considered, but here it's not needed will remove the array and use the equality check.